backend/composer/admin.py

```diff
@@ -231,7 +231,7 @@ class ConnectivityStatementAdmin(
     list_per_page = 10
     # The name of one or more FSMFields on the model to transition
     fsm_field = ("state",)
-    readonly_fields = ("state",)
+    readonly_fields = ("state", "journey_path")
```

Comment:
Since journey_path is a complex object and not the string I think it might make more sense to actually exclude it instead of having it as read-only